content/blog/2016-09-02-promcon-2016-recap.md

```diff
@@ -0,0 +1,152 @@
+---
+title: PromCon 2016 - It's a wrap!
+created_at: 2016-09-02
+kind: article
+author_name: Julius Volz
+---
+
+## What happened
+
+Last week, eighty Prometheus users and developers from around the world came
+together for two days in Berlin for the first-ever conference about the
+Prometheus monitoring system: [PromCon 2016](https://promcon.io/). The goal of
```

Comment:
So I was think, in order to be able to use the same domain for all upcoming promcons, we'll probably need to introduce subpaths. Like https://promcon.io/2016-berlin/ with redirects from the front page to each individual page. Once that's discussed and done, we should update this link here.

No blocker, can be done later.